src/main/java/com/savvato/tribeapp/controllers/ConnectAPIController.java

```diff
@@ -18,48 +16,57 @@
 import org.springframework.messaging.handler.annotation.Payload;
 import org.springframework.web.bind.annotation.*;
 
+import javax.validation.Valid;
+import java.util.Optional;
+
 @RestController
 @Tag(name = "connect", description = "Connections between users")
 @RequestMapping("/api/connect")
 public class ConnectAPIController {
-  @Autowired ConnectService connectService;
+    @Autowired
+    ConnectService connectService;
 
-  ConnectAPIController() {}
+    ConnectAPIController() {
+    }
+
+    @GetQRCodeString
+    @GetMapping("/{userId}")
+    public ResponseEntity getQrCodeString(
+            @Parameter(description = "The user ID of a user", example = "1") @PathVariable Long userId) {
 
-  @GetQRCodeString
-  @GetMapping("/{userId}")
-  public ResponseEntity getQrCodeString(
-      @Parameter(description = "The user ID of a user", example = "1") @PathVariable Long userId) {
+        Optional<String> opt = connectService.storeQRCodeString(userId);
 
-    Optional<String> opt = connectService.storeQRCodeString(userId);
+        if (opt.isPresent()) {
+            return ResponseEntity.status(HttpStatus.OK).body(opt.get());
+        } else {
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(null);
+        }
+    }
 
-    if (opt.isPresent()) {
-      return ResponseEntity.status(HttpStatus.OK).body(opt.get());
-    } else {
-      return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(null);
+    @Connect
+    @PostMapping
+    public boolean connect(@RequestBody @Valid ConnectRequest connectRequest) {
```

Comment:
Lets return a ResponseEntity here as well, rather than just  boolean.

Reply:
I think this is a different endpoint, but I'd be glad to refactor this in another issue! The endpoint in this PR is the one with the @DeleteMapping - this line seems to have changed due to reformatting.